contacts.html

```diff
@@ -24,21 +24,24 @@ <h2>Popular science magazine OYLA</h2>
             <h3>Contact Us</h3>
             <h4>Information about the publisher</h4>
             <address>
-                <p>Popular science magazine OYLA is owned and operated by OYLA MAGAZINE LTD (COMPANY NUMBER: 12718869).<br>128 City Road, London, EC1V 2NX, United Kingdom.<br><br><strong>Tel:</strong> +44 20 4577 04 07<br><strong>Working hours:</strong> 9-5pm, Mon-Fri</p>
+                <p>Popular science magazine OYLA is owned and operated by OYLA MAGAZINE LTD (COMPANY NUMBER: 12718869).<br>128 City Road, London, EC1V 2NX, United Kingdom.<br><br><strong>Tel:</strong> +44 20 4577 04 07<br><strong>Working hours:</strong> 9-5pm, Mon-Fri
```

Comment:
Closing paragraph tag is repositioned and looks good. However content of the paragraph element still needs better indentation